e2e/tests/features/209-rundown-shortcuts.spec.ts

```diff
@@ -1,5 +1,23 @@
 import { test, expect } from '@playwright/test';
 
+test('Edit', async ({ page }) => {
+  await page.goto('http://localhost:4001/rundown');
+
+  // clear rundown
+  await page.getByRole('button', { name: 'Clear rundown' }).click();
+  await page.getByRole('button', { name: 'Delete all' }).click();
+
+  //create event
+  await page.getByRole('button', { name: 'Create Event' }).click();
+  await page.getByTestId('block__title').click();
+  await page.getByTestId('block__title').fill('test');
+  await page.getByTestId('block__title').press('Enter');
+
+  //assert
+  await expect(page.getByTestId('entry-1')).toBeVisible();
+  await expect(page.getByTestId('entry-1').getByTestId('block__title')).toHaveValue('test');
+});
+
```

Comment:
I am not sure this is testing a shortcut or otherwise giving us much?
we already have the `204-editor-crud` covering this. if anything we can improve that test instead